cmd/cainjector/app/start.go

```diff
@@ -126,13 +131,24 @@ func (o *InjectorControllerOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.PprofAddr, "profiler-address", cmdutil.DefaultProfilerAddr, "Address of the Go profiler (pprof) if enabled. This should never be exposed on a public interface.")
 
 	utilfeature.DefaultMutableFeatureGate.AddFlag(fs)
+
+	logf.AddFlags(o.Logging, fs)
+
+	// The controller-runtime flag (--kubeconfig) that we need
+	// relies on the "flag" package but we use "spf13/pflag".
+	var controllerRuntimeFlags flag.FlagSet
+	config.RegisterFlags(&controllerRuntimeFlags)
+	controllerRuntimeFlags.VisitAll(func(f *flag.Flag) {
+		fs.AddGoFlag(f)
+	})
```

Comment:
Why do we not need the same thing in the controller and webhook?

Reply:
Because they do not use controller runtime.